apiserver/allfacades.go

```diff
@@ -123,6 +123,7 @@ func AllFacades() *facade.Registry {
 	reg("Application", 2, application.NewFacade)
 	reg("Application", 3, application.NewFacade)
 	reg("Application", 4, application.NewFacade)
+	reg("Application", 5, application.NewFacade) // UpdateSeries
```

Comment:
Will do.

There is a side question on this.  Version 5 is already used in develop.  What is the correct version to use here?